src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.xml

```diff
@@ -36,6 +36,10 @@
             <tag name="cache.pool" />
         </service>
 
+        <service id="cache.messenger" parent="cache.app" public="false">
```

Comment:
I'm using `cache.app` because we need this value to be shared amongst multiple servers and it should persist through deploys. However, there is no precedent that I know if for the core to use `cache.app`. And also, perhaps the service id should be more specific to the "messenger restart" - and not just `cache.messenger`?

Reply:
Can't we use a bus instead here to broadcast to all workers?

Reply:
If we keep a cache pool, it should be a dedicated one btw, extending from `cache.app`.

Reply:
> Can't we use a bus instead here to broadcast to all workers?

It means we'd have to build a broadcast feature, which we don't have nor plan (yet?) to have.

Reply:
I've made the cache service id more specific - still extends from `cache.app` like before